src/app/shared/utils/fields.utils.ts

```diff
@@ -122,4 +122,35 @@ export class FieldsUtils {
     return Object.assign({}, obj);
   }
 
+  showGrayBar(caseField: CaseField, element: HTMLElement) {
```

Comment:
if you look at the rest of methods in fields.utils.ts, you'll see that this class is not concerned with UI modifications. So a method like `showGrayBar` does not belong here.
I was thinking of introducing a new class display.utils.ts and move it there. But perhaps we can find some other placement for this method which is even better, I'll give it a thought and come back to you